### DISCUSSION TITLE
Not working?  Edit Hyperlink

### DISCUSSION THREAD

**Discussion by bstott** [freeplane/freeplane#1018] [Help] [answered]
Hi, new here.

Windows 10; Freeplane 1.10.6

I am attempting to edit hyperlinks. However, the documented 'right click' > edit > ... doesn't have a hyperlink edit field.  

How to edit hyperlinks?

**Comment by sifran-github** [comment, marked as answer]
You should use the command add link. if a link is already there, you will be able to edit it. Use one of these options:
1. 'right click' > Add link -> Add hyperlink (URL)...
2. Menu: Insert -> Link -> Add hyperlink (URL)...
3. Ctrl+K

Also, if you can't find a command,  use help->command search, and make sure "menu items" is checked.

**Reply by bstott** [reply]
OK. That really isn't intuitive and Sucks as an answer.  But, I see it is an easy secret.

It should be moved to simply a Hyperlink drop down with options. NOT ''' INSERT"""

Thanks.

**Reply by bstott** [reply]
I had used the Ctrl+K and the Command Search. Neither were obvious nor intuitive that Modify was possible.  Because, The Topic was INSERT.   But, I have it now.   The software isn't clear.  Thanks, again.
